lms/djangoapps/certificates/models.py

```diff
@@ -998,6 +999,13 @@ class CertificateTemplate(TimeStampedModel):
         help_text=_(u'On/Off switch.'),
         default=False,
     )
+    language = models.CharField(
+        max_length=2,
+        choices=LANGUAGES,
+        blank=True,
+        null=True,
+        help_text=_(u'Optional. Only certificates for courses in the selected language will be rendered using this template.<br />*Note* Course languages are determined by the first two letters of the language code.')
```

Comment:
Should we mention that to add another languages, one can do so in settings.py?

Reply:
Yeah that is a good question. Is that a Jasper question, or can we just make that decision from a developer standpoint?

Reply:
I think it's safe to just add it

Reply:
Just a sanity check, but does the inclusion of an HTML-tag here imply that we're not escaping HTML from the DB in the Django-admin? That seems like a security risk, especially if we're translating this text.